packages/thumbprint-react/rollup.config.js

```diff
@@ -1,20 +1,26 @@
 import babel from 'rollup-plugin-babel';
 import path from 'path';
 import copy from 'rollup-plugin-cpy';
+import resolve from 'rollup-plugin-node-resolve';
```

Comment:
What changed that made this necessary?

Reply:
i don't remember. some quirk of babel and typescript having different module resolution logic